test/SymbolGraph/Symbols/Mixins/DeclarationFragments/Subheading/Function.swift

```diff
@@ -11,66 +11,66 @@ public func foo<S>(f: @escaping () -> (), ext int: Int = 2, s: S) where S: Seque
 // CHECK: names
 // CHECK-NEXT: "title": "foo(f:ext:s:)"
 // CHECK:      "subHeading": [
```

Comment:
Should we also test attributes on functions; maybe by adding one to this function?

Reply:
I can add a test with an `@objc` attribute on a method, I think that would be a good case.